packages/reports/addon/consumers/request/filter.js

```diff
@@ -73,9 +73,9 @@ export default ActionConsumer.extend({
      * @param {Object} route - route that has a model that contains a request property
      * @param {Object} dimension - dimension to filter
      */
-    [RequestActions.ADD_DIM_FILTER]: ({ currentModel }, dimension) => {
+    [RequestActions.ADD_DIM_FILTER]: ({ currentModel }, dimension)   => {
       get(currentModel, 'request').addFilter(
-        assign({ dimension }, DEFAULT_DIM_FILTER)
+        assign({ dimension}, DEFAULT_DIM_FILTER, {field: get(dimension, 'primaryKeyFieldName') || 'id'})
```

Comment:
`primaryKeyFieldName` already defaults to `id`.
https://github.com/yahoo/navi/blob/fc6bb19c3b376c9309f49911aa1f79c6291e3a65/packages/data/addon/models/metadata/dimension.js#L87

Reply:
Does it make sense to have `DEFAULT_DIM_FILTER` now?

Reply:
So doing that assumes that the consumer is a dimension object, and in practice it seems that it is from where I tested it. However, a lot of tests pass in just the dimension name.  Is this bad mocks? or will we actually run into dimension simply being a string?

Reply:
Bad mocks I hope. I might be time to consider propTypes or flow 😬